### PR TITLE
Delete old diet study screens

### PR DESCRIPTION
**Pull request by selvach**
- Deletes the old DietStudy (v1) screens
- Deletes the DietStudyEmailModal which was in the initial v2 designs, but later dropped. 
- Add US internationalisation
- Shows different Avatars based on UK vs US flows

### CODE REVIEW

File: assets/lang/en-US.json [hunk 1]
```diff
@@ -84,6 +69,14 @@
     }
   },
 
+  "diet-study": {
```
Comment (bionara) [inline assets/lang/en-US.json:72]: any reason this is added? just house keeping?

File: assets/lang/en-US.json [hunk 2]
```diff
@@ -84,6 +69,14 @@
...
+    "doctor-name": "Dr Sarah Berry",
+    "doctor-title": "Nutritional Scientist",
+    "doctor-location": "King’s College London",
+    "gut-tips-body-1": "Fiber is fundamental - choose fiber-rich foods like wholegrains, beans, pulses, fruit, vegetables, nuts and seeds.",
```
Comment (bionara) [inline assets/lang/en-US.json:76]: i think the americans spell fiber as fibre?

**Review by bionara — APPROVED** [review]
left a Q about US spelling

File: assets/lang/en-US.json (hunk 2 above)
Comment (selvach) [reply assets/lang/en-US.json:76]: UK spelling: fibre
US spelling: fiber

**Review by bionara — APPROVED** [review]

**Pull request merged**